Makefile

```diff
@@ -77,11 +77,16 @@ test: ## Run the unit tests (used in the CI)
 
 .PHONY: test-coverage
 test-coverage:  ## Run coveralls
-	# remove all coverage files if exists
+	# remove all coverage files if existss
```

Comment:
revert ^

Reply:
oof, good catch!

Can you tell I have one of the awful scissor apple keyboards? :)